[2025-08-18 21:00:04,406] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_python.Python_start scheduled__2025-08-18T00:00:00+00:00 [queued]>
[2025-08-18 21:00:04,412] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_python.Python_start scheduled__2025-08-18T00:00:00+00:00 [queued]>
[2025-08-18 21:00:04,412] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 21:00:04,413] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-18 21:00:04,413] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 21:00:04,427] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): Python_start> on 2025-08-18 00:00:00+00:00
[2025-08-18 21:00:04,430] {standard_task_runner.py:52} INFO - Started process 2215592 to run task
[2025-08-18 21:00:04,435] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag_python', 'Python_start', 'scheduled__2025-08-18T00:00:00+00:00', '--job-id', '184', '--raw', '--subdir', 'DAGS_FOLDER/dag_Python.py', '--cfg-path', '/tmp/tmpmmo_g2vr', '--error-file', '/tmp/tmpt5u6kls_']
[2025-08-18 21:00:04,435] {standard_task_runner.py:80} INFO - Job 184: Subtask Python_start
[2025-08-18 21:00:04,474] {task_command.py:370} INFO - Running <TaskInstance: dag_python.Python_start scheduled__2025-08-18T00:00:00+00:00 [running]> on host itoneptl
[2025-08-18 21:00:04,520] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_python
AIRFLOW_CTX_TASK_ID=Python_start
AIRFLOW_CTX_EXECUTION_DATE=2025-08-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-18T00:00:00+00:00
[2025-08-18 21:00:04,521] {logging_mixin.py:115} INFO - Hello!!!
[2025-08-18 21:00:04,521] {python.py:173} INFO - Done. Returned value was: None
[2025-08-18 21:00:04,527] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dag_python, task_id=Python_start, execution_date=20250818T000000, start_date=20250819T000004, end_date=20250819T000004
[2025-08-18 21:00:04,566] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 21:00:04,582] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
